draft/spec/index.html

```diff
@@ -861,7 +861,32 @@ <h2>Logs Directory</h2>
                 wrong, while others may wish to only store a log entry if an intervention was required.
             </p>
         </blockquote>
+    </section>
 
+    <section id="object-extensions">
+        <h2>Object Extensions</h2>
+        <p>
+            The base directory of an OCFL Object MAY contain a directory named <code>extensions</code> for the purposes
+            of extending the functionality of an OCFL Object. The <code>extensions</code> directory MUST NOT contain
+            any files, and no sub-directories other than extension sub-directories. Extension sub-directories MUST be
```

Comment:
Per https://github.com/OCFL/spec/wiki/2020.01.21-Editors-Meeting I think agreement was that:
  * "Extensions SHOULD be a defined/registered"
  * "Extensions MUST be in their own sub-directory"